classes/services/queryBuilders/PKPUserQueryBuilder.inc.php

```diff
@@ -150,6 +153,21 @@ public function filterByRoleIds($roleIds) {
 		return $this;
 	}
 
+	/**
+	 * Set user group filter
+	 *
+	 * @param $userGroupIds int|array
+	 *
+	 * @return \PKP\Services\QueryBuilders\PKPUserQueryBuilder
+	 */
+	public function filterByUserGroup($userGroupIds) {
+		if (!is_null($userGroupIds) && !is_array($userGroupIds)) {
```

Comment:
I'm bikeshedding here, but a more concise way of doing this would be an explicit `(array)` typecast.

https://www.php.net/manual/en/language.types.array.php#language.types.array.casting:
> In other words, (array)$scalarValue is exactly the same as array($scalarValue).

If `null` values are allowed, the self-documentation should reflect that. (Note that `(array) null === []` so an explicit cast might be useful there too.)

Reply:
Hmm, this method has been introduced 2 days ago by another PR, so I'll just merge it.